### PR TITLE
Fix octavia network attachment subnet range.

### PR DESCRIPTION
**Pull request by beagles**
Fixes the expected subnet IP range for the octavia network attachment. A previous change erroneously set this to the range for the tenant network instead of the provider network.

### CODE REVIEW

**Review by karelyatin — COMMENTED** [review]
/lgtm

@psathyan likely you pointing out this bit earlier over slack?

File: examples/dt/bgp/control-plane/nncp/values.yaml [hunk 1]
```diff
@@ -189,9 +189,9 @@ data:
         "bridge": "octbr",
         "ipam": {
           "type": "whereabouts",
-          "range": "172.24.0.0/24",
-          "range_start": "172.24.0.30",
-          "range_end": "172.24.0.70"
+          "range": "172.23.0.0/24",
```
Comment (psathyan) [inline examples/dt/bgp/control-plane/nncp/values.yaml:192]: Can we use user provider CIDR? If no, I would need to change in the downstream configurations that we pass.

Comment (psathyan) [reply examples/dt/bgp/control-plane/nncp/values.yaml:192]: Thank you @beagles for the clarification.

**Review by psathyan — APPROVED** [review]

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
Has this been tested?

**Comment by psathyan** [comment]
> Has this been tested?

The `uni01alpha` downstream jobs have already picked up the change and have executed successfully in the last 2 runs.

**Pull request merged**